.travis.yml

```diff
@@ -73,13 +75,6 @@ matrix:
       - dist: trusty
         env:
           - JOB="3.6, slow" ENV_FILE="ci/travis-36-slow.yaml" SLOW=true
-      - dist: trusty
```

Comment:
is there a reason you are removing this job?

Reply:
I'm removing it from the allow_failures section.

Reply:
oh ok, good